smart-contracts/test/Lock/Lock.js

```diff
@@ -43,13 +45,11 @@ contract('Lock', (accounts) => {
         numberOfOwners = new BigNumber(numberOfOwners)
         assert.strictEqual(owner, accounts[0])
         assert.equal(expirationDuration.toFixed(), 60 * 60 * 24 * 30)
-        assert.strictEqual(
-          Units.convert(keyPrice, 'wei', 'eth'),
-          '0.01'
-        )
+        assert.strictEqual(Units.convert(keyPrice, 'wei', 'eth'), '0.01')
         assert.equal(maxNumberOfKeys.toFixed(), 10)
         assert.equal(outstandingKeys.toFixed(), 0)
         assert.equal(numberOfOwners.toFixed(), 0)
+        assert(publicLockVersion.eq(0)) // needs updating each lock-version change
```

Comment:
Minor style:
Maybe use the `assert.equal` style from the line above.  I had previously implemented asserts using `assert(var.eq(other))` but the trouble is when the assert fails the error message is not helpful.  Using `toFixed()` makes a string out of the number so when the `assert.equal` fails it prints the actual values, which is helpful while debugging.

Reply:
sounds good.